kobo/client/__init__.py

```diff
@@ -111,6 +109,14 @@ def set_hub(self, username=None, password=None):
         self.hub = HubProxy(conf=self.conf)
 
 
+class ClientCommandContainer(BaseClientCommandContainer):
+    """A general-purpose subclass of BaseClientCommandContainer that loads configurations immediately at instantiation"""
+    def __init__(self, conf, **kwargs):
+        super().__init__()
```

Comment:
Hmm, I think I mentioned earlier that super() with no args doesn't work on python2 (and you do want this to be compatible with python2).

The fact that CI is not failing seems to indicate that this code isn't covered by a test, and the coverage report confirms it:
https://coveralls.io/builds/35010530/source?filename=kobo%2Fclient%2F__init__.py#L115

Could you please ensure it's covered at least to the point of "does not crash"?

Reply:
@rohanpm I think these latest changes should address python2 compatibility.